.github/workflows/build_stable.yml

```diff
@@ -75,18 +75,20 @@ jobs:
         run: python3 ./.github/scripts/get_latest_tags.py --repo erpnext --version 13
 
       - name: Update
-        run: python3 ./.github/scripts/update_example_env.py
+        run: |
```

Comment:
Change step name from `update_example_env` to more generic one

Reply:
reuse same function?

Reply:
Just the name: https://github.com/frappe/frappe_docker/blob/dfd98dd1216dda1241b31743639a118ecf8b456d/.github/workflows/build_stable.yml#L60

Reply:
updated